pos_order_load/README.rst

```diff
@@ -0,0 +1,38 @@
+POS Order Load and Save
+=======================
+
+This module allows to load existing POS order.
+In this version, when the loaded order
+is validated in the POS, a new one is created.
```

Comment:
Please indicate here that you can also save (I prefer this word instead of backup) the order, and then recover/load it.